packages/gasket-plugin-cypress/README.md

```diff
@@ -0,0 +1,28 @@
+# @gasket/plugin-cypress
+
+Creates a `cypress` based testing environment for your Gasket application.
+
+## Installation
+
+This plugin should only be used during the create command for new apps.
+
+```
+gasket create <app-name> --plugins @gasket/plugin-cypress
+```
+
+## Usage
+
+When you create a new gasket application that is configured with the `cypress`
+plugin it will prepare it with a `cypress` based testing environment. It will add
+the following `scripts` to the `package.json`:
+
+NOTE: before running `npm run cypress` or `npm run cypress:headless`, you should first start the server in a separate tab.
```

Comment:
You should also include a note about requiring `gasket build` to have been run before trying to start the server or using the e2e scripts.